### ISSUE TITLE
Questions about Your Paper

### ISSUE THREAD

**Issue by wangguanzhi** [VisualComputingInstitute/diffusion-e2e-ft#5]
Hi Team,

I am truly impressed by the outstanding work you’ve done! Congrats on the great paper! I have a few questions about your paper:
1. For Sec 3.2, you pointed out a flaw in the Marigold inference scheduler implementation. Could you please point me to which part in the https://github.com/prs-eth/Marigold codebase and share your code patch?
2. In Sec. 4, you set the noise to zero and fix t=T without sampling. Does this mean that the training essentially becomes a standard supervised learning that maps RGB to depth directly, but going through SD's latent space? Have I missed anything?
3. I would really appreciate it if you could share your training code! Do you have an ETA?

Thanks so much for your time!

**Comment by kabouzeid** [comment]
Thank you for the kind words :)

1. Our inference code is copied and modified from Marigold, setting `timestep_spacing="leading"` is the important fix. https://github.com/VisualComputingInstitute/diffusion-e2e-ft/blob/3c02cd11d59d82e9002cce40c48f1a3dca72d319/Marigold/run.py#L275
2. Yes. Hence the title of the paper "fine-tuning ... is easier than you think"
3. Training code is coming soon.
